hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/test/LambdaTestUtils.java

```diff
@@ -823,6 +922,19 @@ public static <E extends Throwable> E verifyCause(
     }
   }
 
+  private static String toString(Collection<String> strings) {
```

Comment:
java8 has a couple of ways to do this; StringJoiner or Collections:
https://www.baeldung.com/java-list-comma-separated-string

Reply:
I added a change that uses the Java stream collector.